### PR TITLE
command line docs

### PR DESCRIPTION
**Pull request by ben-bay**
Attempts to complete the page of the web docs on command line options.

@koning is this the type of update you were hoping for?

### CODE REVIEW

File: docs/source/merlin_commands.rst [hunk 1]
```diff
@@ -235,47 +235,44 @@ with workers, such as via ``merlin stop-workers --workers``.
 
 .. _stop-workers:
 
-Stopping workers (stop-workers)
--------------------------------
+Stopping workers (``merlin stop-workers``)
+------------------------------------------
```
Comment (koning) [inline docs/source/merlin_commands.rst]: This also needs --task_server

File: docs/source/merlin_commands.rst [hunk 2]
```diff
@@ -302,8 +299,8 @@ If the specified directory does not exist Merlin will automatically create it.
 This will generate the example workflow at the specified location, ready to be run.
 
 
-Purging Tasks (purge)
----------------------
+Purging Tasks (``merlin purge``)
```
Comment (koning) [inline docs/source/merlin_commands.rst]: purge also has --vars

File: docs/source/merlin_commands.rst [hunk 3]
```diff
@@ -322,33 +319,44 @@ task server, run:
...
 
 .. code:: bash
 
     $ merlin purge <input.yaml> --steps step1 step2
 
 
-Debug Info
-----------
-More information can be output by increasing the logging level
+Status (``merlin status``)
```
Comment (koning) [inline docs/source/merlin_commands.rst]: status has --task_server

File: docs/source/merlin_commands.rst (hunk 1 above)
Comment (ben-bay) [reply docs/source/merlin_commands.rst]: The reason I excluded task server is that the way merlin currently works, the only possible task server is celery. So I thought removing that choice from the docs might help users from getting confused.

(Any opinion on this @lucpeterson?)

File: docs/source/merlin_commands.rst (hunk 3 above)
Comment (ben-bay) [reply docs/source/merlin_commands.rst]: My reasoning here is same as for `stop-workers`

File: docs/source/merlin_commands.rst (hunk 2 above)
Comment (ben-bay) [reply docs/source/merlin_commands.rst]: Added.

File: docs/source/merlin_commands.rst (hunk 1 above)
Comment (koning) [reply docs/source/merlin_commands.rst]: Yes, that is a good reason.  I only mention it for completeness, if they have a  question about the arguments for ``merlin stop-workers -h`` where will they that information?

Comment (ben-bay) [reply docs/source/merlin_commands.rst]: What I'll do for now is add a section about this, and mention that celery is the only option. We might consider deprecating this flag later on.

Comment (koning) [reply docs/source/merlin_commands.rst]: Sounds good.

**Pull request merged**